core/secrets.py

```
@@ -0,0 +1,574 @@
+        )
+
+        configs["environment"] = SecretConfig(
+            key="ENVIRONMENT",
+            default="development",
+            required=False
+        )
+
+        # API Configuration
+        configs["api_host"] = SecretConfig(
+            key="API_HOST",
+            default="0.0.0.0",
```

Ruff (0.14.10)

Possible binding to all interfaces

(S104)